radiation/python/stencils_radiation_driver.py

```diff
@@ -0,0 +1,1073 @@
+		 externals={
+				   "con_epsq" : con_epsq,
+                   }
+		)
+def cloud_comp_5(ccnd : Field[(DTYPE_FLT, (4,))],
+				 tracer1 : Field[(DTYPE_FLT,(8,))],
+				 tracer_idx_0 : int,
+				 tracer_idx_1 : int,
+				 tracer_idx_2 : int,
+				 tracer_idx_3 : int,
+				 tracer_idx_4 : int,
+				):
+	from __externals__ import(con_epsq)
+	with computation(PARALLEL), interval(1,None):
+		ccnd[0,0,0][0] = tracer1[0,0,0][tracer_idx_0]
+		ccnd[0,0,0][1] = tracer1[0,0,0][tracer_idx_1]
+		ccnd[0,0,0][2] = tracer1[0,0,0][tracer_idx_2]
+		ccnd[0,0,0][3] = tracer1[0,0,0][tracer_idx_3] + tracer1[0,0,0][tracer_idx_4]
+
+		if ccnd[0,0,0][0] < con_epsq:
+			ccnd[0,0,0][0] = 0.0
+		if ccnd[0,0,0][1] < con_epsq:
+			ccnd[0,0,0][1] = 0.0
+		if ccnd[0,0,0][2] < con_epsq:
+			ccnd[0,0,0][2] = 0.0
+		if ccnd[0,0,0][3] < con_epsq:
+			ccnd[0,0,0][3] = 0.0
+
+@stencil(backend=backend)
+def cloud_comp_5_v2(ccnd : Field[(DTYPE_FLT, (1,))],
+				 tracer1 : Field[(DTYPE_FLT,(8,))],
+				 tracer_idx_0 : int,
+				 tracer_idx_1 : int,
+				 tracer_idx_2 : int,
+				 tracer_idx_3 : int,
+				 tracer_idx_4 : int,
+				 EPSQ : float,
+				):
+	with computation(PARALLEL), interval(1,None):
+		ccnd[0,0,0][0] = tracer1[0,0,0][tracer_idx_0] + tracer1[0,0,0][tracer_idx_1] + \
+						 tracer1[0,0,0][tracer_idx_2] + tracer1[0,0,0][tracer_idx_3] + \
+						 tracer1[0,0,0][tracer_idx_4]
+
+
+@stencil(backend=backend)
+def ccnd_zero(ccnd : Field[(DTYPE_FLT, (1,))],
+ 			  EPSQ : float,
+			 ):
+	with computation(PARALLEL), interval(1,None):
+		if ccnd[0,0,0][0] < EPSQ:
+			ccnd[0,0,0][0] = 0.0
+
+@stencil(backend=backend)
+def cloud_cover(cldcov : FIELD_FLT,
+				effrl : FIELD_FLT,
+				effri : FIELD_FLT,
+				effrr : FIELD_FLT,
+				effrs : FIELD_FLT,
+				phy_f3d : Field[(DTYPE_FLT,(8,))],
+				tracer1 : Field[(DTYPE_FLT,(8,))],
+				indcld_idx : int, 
+				ntclamt_idx : int,							
+				uni_cld : bool,
+				effr_in : bool,
+				imp_physics : DTYPE_INT,
+				):
+	with computation(PARALLEL), interval(1,None):
+		if uni_cld:
+			cldcov = phy_f3d[0,0,0][indcld_idx]
+			if effr_in:
+				effrl = phy_f3d[0,0,0][1]
+				effri = phy_f3d[0,0,0][2]
+				effrr = phy_f3d[0,0,0][3]
+				effrs = phy_f3d[0,0,0][4]
+
+		elif imp_physics == 11:
+			cldcov = tracer1[0,0,0][ntclamt_idx]
+			if effr_in:
+				effrl = phy_f3d[0,0,0][0]
+				effri = phy_f3d[0,0,0][1]
+				effrr = phy_f3d[0,0,0][2]
+				effrs = phy_f3d[0,0,0][3]
+		else:
+			cldcov = 0.0
+
+@stencil(backend=backend)
+def add_cond_cloud_water(deltaq : FIELD_FLT,
+						 cnvw   : FIELD_FLT,
+						 cnvc   : FIELD_FLT,
+						 cldcov : FIELD_FLT,
+						 effrl : FIELD_FLT,
+						 effri : FIELD_FLT,
+						 effrr : FIELD_FLT,
+						 effrs : FIELD_FLT,
+						 phy_f3d : Field[(DTYPE_FLT,(8,))],
+						 ccnd : Field[(DTYPE_FLT, (1,))],
+						 num_p3d : DTYPE_INT,
+						 npdf3d  : DTYPE_INT,
+						 ncnvcld3d : DTYPE_INT,
+						 imp_physics : DTYPE_INT,
+						 lextop : bool,
+						 ivflip : int,
+						 effr_in : bool,
```

Comment:
I think these are set from the namelist and as a result won't change during the simulation? Would be better to pass in as externals